src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputResources.java

```diff
@@ -16,12 +16,26 @@
 
 import com.google.gwt.resources.client.ClientBundle;
 import com.google.gwt.resources.client.CssResource;
+import com.google.gwt.resources.client.ImageResource;
 
 public interface FindOutputResources extends ClientBundle
 {
+   @Source("ExpandReplaceIcon_2x.png")
+   ImageResource expandReplaceIcon2x();
+
+   @Source("CollapseReplaceIcon_2x.png")
+   ImageResource collapseReplaceIcon2x();
```

Comment:
These icons don't appear to be used any more so should be removed.

Reply:
Fixed